scene/2d/tile_map.h

```diff
@@ -116,7 +116,7 @@ struct TileMapQuadrant {
 	List<RID> occluders;
 
 	// Physics.
-	List<RID> bodies;
+	Map<Vector2i, RID> bodies;
```

Comment:
Yes, it's used when updating the bodies transform (see use of `body_element.key`)

Reply:
I see, so it's used for the key/value pair but not for the lookup function.

In this case, is it possible to use only one map `Map<RID, Vector2i>` since you do use lookup from RID to coord but not the other way around, instead of having two maps?

Reply:
That makes sense, I updated the PR so that it retrieves the coords from the body instead.